src/ImageSharp/Formats/Jpeg/JpegEncoderCore.cs

```diff
@@ -822,11 +822,12 @@ private void WriteStartOfFrame(int width, int height, int componentCount)
             {
                 for (int i = 0; i < componentCount; i++)
                 {
-                    this.buffer[(3 * i) + 6] = (byte)(i + 1);
+                    int i3 = 3 * i;
```

Comment:
Maybe you could just increment `i3` instead of doing the multiplication? You then also don't need the i3+6/7/8?

Reply:
I thought about that first, but i is used itself in liness 829 and 830 again.
We could store the iterations initial i as iPrime or whatever, or store the subsamples[i] and chroma[i] first, but it doesn't make it better,
so I decided to keep it as less invasive as possible and would like not to do more in this PR alone.
Feel free to add further improvements.